presto-docs/src/main/sphinx/presto_cpp/hive-properties-session.rst

```diff
@@ -0,0 +1,16 @@
+=============================
+Presto C++ Hive Session Properties
+=============================
+
+Session Properties
+------------------
+
+The following are the native session properties for Hive connector on C++ based Presto.
+
+``native_stats_based_filter_reorder_disabled``
```

Comment:
Is this the only one for Hive connector?

Reply:
yes